x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/multi_metric_view/metric_selection.tsx

```diff
@@ -33,7 +33,7 @@ export const MultiMetricDetectors: FC<Props> = ({ setIsValid }) => {
 
   const jobCreator = jc as MultiMetricJobCreator;
 
-  const { fields } = newJobCapsService;
+  const [fields] = useState([...newJobCapsService.fields, ...jobCreator.runtimeFields]);
```

Comment:
Could the combined list of fields be sorted for display? Currently the runtime fields are all displayed at the end:

![image](https://user-images.githubusercontent.com/7405507/107958957-a5aa9a00-6f9a-11eb-8607-a7fa7f0971ba.png)

Reply:
Fixed in c1f850a06ecaafe3bb99740d8b069e86fb448dbd